API Extractor cannot find name OffscreenCanvas

Hi, I'm trying to publish a TypeScript library npm package that includes Pixi, and Microsoft API Extractor is failing `OffscreenCanvas`

```
build:types
> tsc -p ./tsconfig.json --outDir build --declaration true && api-extractor run

node_modules/@pixi/core/index.d.ts:1543:45 - error TS2304: Cannot find name 'OffscreenCanvas'.

1543     static test(source: unknown): source is OffscreenCanvas | HTMLCanvasElement;
                                                 ~~~~~~~~~~~~~~~

node_modules/@pixi/text/index.d.ts:272:41 - error TS2304: Cannot find name 'OffscreenCanvas'.

272     static _canvas: HTMLCanvasElement | OffscreenCanvas;
                                            ~~~~~~~~~~~~~~~

node_modules/@pixi/text/index.d.ts:273:49 - error TS2304: Cannot find name 'OffscreenCanvasRenderingContext2D'.

273     static _context: CanvasRenderingContext2D | OffscreenCanvasRenderingContext2D;
                                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@pixi/text/index.d.ts:300:105 - error TS2304: Cannot find name 'OffscreenCanvas'.

300     static measureText(text: string, style: TextStyle, wordWrap?: boolean, canvas?: HTMLCanvasElement | OffscreenCanvas): TextMetrics_2;
                                                                                                            ~~~~~~~~~~~~~~~
```

Think there's some movement with offscreen canvas, maybe this has already been addressed.  

FYI, or please advise if this seems in error.


### Environment

- **pixi.js**:  6.2.2
- **tslib**: 2.3.1
- **TypeScript**: 4.5.5
- **@microsoft/api-extractor**: 7.19.4
- **OS & Version**: macOS Monterey, Apple M1 Pro

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -28,6 +28,9 @@
     "type": "opencollective",
     "url": "https://opencollective.com/pixijs"
   },
+  "dependencies": {
+    "@types/offscreencanvas": "^2019.6.4"
+  },
   "peerDependencies": {
     "@pixi/constants": "6.2.1",
     "@pixi/math": "6.2.1",

